platforms/src/ZkSync/__tests__/accountAnalysis.test.ts

```diff
@@ -0,0 +1,90 @@
+import { RequestPayload } from "@gitcoin/passport-types";
+import axios from "axios";
+import {
+  ZkSyncScore20Provider,
+  ZkSyncScore50Provider,
+  ModelResponse,
+  getZkSyncAnalysis,
+  ZkSyncScore5Provider,
+} from "../Providers/accountAnalysis";
+
+const mockAddress = "0x0";
+let mockContext = {};
+const mockResponse = (score: number): { data: ModelResponse } => ({
+  data: {
+    data: {
+      human_probability: score,
+    },
+  },
+});
+
+jest.mock("axios");
+const mockedAxios = axios as jest.Mocked<typeof axios>;
+
+describe("AccountAnalysis Providers", () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+    mockContext = {};
+  });
+
+  it("should validate inputs for ZkSyncScore5Provider", async () => {
+    const mockedResponse = mockResponse(5);
+    mockedAxios.post.mockResolvedValueOnce(mockedResponse);
+
+    const ethAdvocateProvider = new ZkSyncScore5Provider();
+    const payload = await ethAdvocateProvider.verify({ address: mockAddress } as RequestPayload, mockContext);
+
+    expect(payload.valid).toBe(true);
+    expect(payload.record).toEqual({ address: mockAddress });
+  });
+
+  it("should validate inputs for ZkSyncScore20Provider", async () => {
+    const mockedResponse = mockResponse(50);
+    mockedAxios.post.mockResolvedValueOnce(mockedResponse);
+    const ethAdvocateProvider = new ZkSyncScore20Provider();
+    const payload = await ethAdvocateProvider.verify({ address: mockAddress } as RequestPayload, mockContext);
+
+    expect(payload.valid).toBe(true);
+    expect(payload.record).toEqual({ address: mockAddress });
+  });
+
+  it("should validate inputs for ZkSyncScore50Provider", async () => {
+    jest.clearAllMocks();
```

Comment:
same as in other places, please put the `jest.clearAllMocks();` this in an `afterEach`